quickstart/views.py

```diff
@@ -35,3 +35,11 @@ class UserViewSet(mixins.RetrieveModelMixin, viewsets.GenericViewSet):
     serializer_class = UserSerializer
     permission_classes = [IsAdminUser]
 
+
+class UserFilterView(APIView):
+    def get(self, request):
+        name = request.GET.get('name', '')
```

Comment:
```suggestion
        name = request.GET.get('name')
```
get - by default if not get anything returns None, no need for empty string